mmengine/runner/log_processor.py

```diff
@@ -236,12 +252,42 @@ def get_log_after_epoch(self,
         cur_loop = self._get_cur_loop(runner, mode)
         dataloader_len = len(cur_loop.dataloader)
 
-        custom_cfg_copy = self._parse_windows_size(runner, batch_idx)
+        custom_cfg_copy = copy.deepcopy(self.custom_cfg)
+        # remove prefix
+        custom_keys = [
+            self._remove_prefix(mode, cfg['data_src'])
+            for cfg in self.custom_cfg
+        ]
+
+        # Count the averaged time and data_time by epoch
```

Comment:
Is this necessary?

Reply:
Counting the average evaluation time may be helpful for those tasks blocked by evaluation. If there is no BC problem, it should not be annoying.